### DISCUSSION TITLE
How to specify a matcher property from command line?

### DISCUSSION THREAD

**Discussion by gwangmu** [GumTreeDiff/gumtree#321] [Q&A]
I referred to two pages ([this issue in 2018](https://github.com/GumTreeDiff/gumtree/issues/79) and [this page about command-line options](https://github.com/GumTreeDiff/gumtree/wiki/Commands)), but none of the command-line combinations seems to work. :disappointed: 

I got one command line to work without generating Java exceptions, but the verbose message seems that the property wasn't properly registered.

> gumtree webdiff -C -M cd_structsim1 path/to/src1 path/to/src2 -v

Could you help me try different matcher properties from the command line? Thank you in advance. :smile:

**Comment by jrfaller** [comment]
Hi! Sorry, I don't know why but I did not see the question :-(

Normally it should be something like `gumtree webdiff path/to/src1 path/to/src2 -M cd_structsim1 0.6` ?
